src/utils/apiClient.js

```diff
@@ -55,17 +55,15 @@ class WrappedClient extends Client {
     if (callback) requestPromise.then(callback, callback);
 
     return requestPromise;
-  };
+  }
 
   resolveFileFromPointers(pointers) {
-    var iteration = 0;
-    var chunks = [];
-    var self = this;
+    const chunks    = [];
```

Comment:
Let's go ahead and remove all aligned assignment spacing as we go through the project over time; it's proving awkward to maintain as code changes, even if it's pretty to view.